CHANGELOG.md

```diff
@@ -77,6 +77,18 @@ All notable changes for each version of this project will be documented in this
     ```
 
 
+### General:
+- `IgxSliderComponent`
+    - **Breaking Change** - The following outputs are renamed:
+        - `onValueChange` to `valueChange`
```

Comment:
Keep in mind `-Change` suffixed events are by convention used for two-way bind in Angular coupled with a `value` property (I believe the slider has one). Is that the intended goal? Because in that case the emit must be typed to the value and it currently uses its own interface. Hey, the `-ed` event uses just the value though 😃

Alternatively, if this should be consistent w/ other event pairs  (-ing/-ed) the first event is usually cancellable and this one is not. Kind of confuses me what use are the two events are intended for based on their names, so while dealing with the naming convention, might as well consider improving the names since we are migrating anyway.

Reply:
`onValueChanged` is renamed to `dragFinished`, as for `valueChange` you and @Aleksandyr already had a discussion about it, and it was decided to leave it as is